.github/workflows/deploy-on-release.yml

```diff
@@ -28,182 +32,116 @@ jobs:
             python_version: 3.8, python_full_version: 3.8.6, python_patch_file: "Modules/Setup",
             python_tarball: "https://www.python.org/ftp/python/3.8.6/Python-3.8.6.tgz"
           }
+        - {
+            name: "Many Linux 2014 Py39", cc: "cl", cxx: "cl",
+            python_version: 3.9, python_full_version: 3.9.5, python_patch_file: "Modules/Setup",
+            python_tarball: "https://www.python.org/ftp/python/3.9.5/Python-3.9.5.tgz"
+          }
     steps:
     - name: Download source code
       shell: bash
       run: |
-        downloads_dir=/home/downloads
-        work_dir=/home/work
-        src_dir=/home/src
-
-        mkdir $downloads_dir
-        mkdir $work_dir
-        mkdir $src_dir
-
+        mkdir $DOWNLOADS_DIR
+        mkdir $WORK_DIR
+        mkdir $SRC_DIR
         # fetch libraries
-        git clone https://github.com/cellml/libcellml -b release --depth 1 $src_dir/libcellml
-        git clone https://github.com/OpenCMISS-Dependencies/zlib -b v1.2.3 --depth 1 $src_dir/zlib
-        git clone https://github.com/OpenCMISS-Dependencies/libxml2 -b v2.9.6 --depth 1 $src_dir/libxml2
-
-        curl -L https://cfhcable.dl.sourceforge.net/project/swig/swig/swig-4.0.2/swig-4.0.2.tar.gz > $downloads_dir/swig-4.0.2.tar.gz
-        curl -L ${{ matrix.config.python_tarball }} > $downloads_dir/Python.tar.gz
-        curl -L https://www.openssl.org/source/openssl-1.1.1h.tar.gz > $downloads_dir/openssl-1.1.1h.tar.gz
-        curl -L ftp://sourceware.org/pub/libffi/libffi-3.2.1.tar.gz > $downloads_dir/libffi-3.2.1.tar.gz
+        git clone https://github.com/hsorby/l1bcellml -b develop --depth 1 $SRC_DIR/libcellml
```

Comment:
should this be `cellml/libcellml` (i.e., the prime libcellml repo)?

Reply:
Definitely.